### ISSUE TITLE
locust doesn't escape the double quotes in the csv output 

### ISSUE THREAD

**Issue by tobeczm** [locustio/locust#1417]
### Describe the bug
```
    @task(1)
    def get(self, size):
        self.client.get("/cash/{}".format(size), name='{"scenario":"get cashes","path":"/cash/[amount]","arguments":[{"size":1}]}'))

```

### Expected behavior
the quotes should be escaped as below:
```
"Type","Name",...
"GET","{""scenario"": ""get cashes"", ""path"": ""/cash/[amount]"", ""arguments"": [{""size"": 1}]}", ...
```

### Actual behavior
in the csv output is looks as below, the quotes are not escaped.

```
"Type","Name",...
"GET","{"scenario": "get cashes", "path": "/cash/[amount]", "arguments": [{"size": 1}]}", ...
```

### Environment

- OS:
- Python version:3.8
- Locust version: 1.0.2
- Locust command line that you ran:
- Locust file contents (anonymized if necessary):

**Comment by heyman** [comment]
Thanks for reporting!

We should consider using the python builtin `csv` module for generating the CSV data.

**Comment by cyberw** [comment]
fixed by #1428!
